nix-mineral.nix

```diff
@@ -490,10 +490,10 @@ in
     })
 
     (l.mkIf cfg.overrides.desktop.hideproc-ptraceable {
-      boot.specialFileSystems."/proc" = l.mkForce {
-        fsType = l.mkDefault "proc";
-        device = l.mkDefault "proc";
-        options = l.mkDefault [
+      boot.specialFileSystems."/proc" = {
+        fsType = l.mkForce "proc";
+        device = l.mkForce "proc";
```

Comment:
IMO, we should avoid setting values if it's for using the same value already defined by nixos:
https://github.com/NixOS/nixpkgs/blob/d49da4c08359e3c39c4e27c74ac7ac9b70085966/nixos/modules/tasks/filesystems.nix#L446-L450
As for `options` maybe we could try to use only list default merge behavior and not use `mkForce`, but that would be a refactoring probably out-of-scope for this PR.

Reply:
Fix it!